aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java

```diff
@@ -438,9 +448,11 @@ public static ResourceModel translateDbClusterFromSdk(
                 .engine(dbCluster.engine())
                 .engineMode(dbCluster.engineMode())
                 .engineVersion(dbCluster.engineVersion())
+                .manageMasterUserPassword(dbCluster.masterUserSecret() != null)
```

Comment:
This is translate function from the object returned by RDS API back into CFN model. RDS API returns master user secret only in case when the cluster/instance is opted into SM integration.